lib/usd/translators/shading/mtlxFileTextureWriter.cpp

```diff
@@ -292,11 +293,29 @@ void MtlxUsd_FileWriter::Write(const UsdTimeCode& usdTime)
         return;
     }
 
-    std::string fileTextureName(fileTextureNamePlug.asString(&status).asChar());
+    const MString rawTextureName = fileTextureNamePlug.asString(&status);
     if (status != MS::kSuccess) {
         return;
     }
 
+    // Resolve texture path à la file node
```

Comment:
I know this is somewhat trivial and maybe unimportant, but that "à" requires that the file encoding be known. I suppose most OS and compilers are fine and assume UTF-8, but I'd rather we keep the contant ASCII :)

Reply:
Oops you're right, well spotted. Fixed in [bebbe7e](https://github.com/Autodesk/maya-usd/pull/3769/commits/bebbe7ea35aaf40edf26a562fce843b269afff84).